core/chains/evm/config/chain_specific_config.go

```diff
@@ -149,6 +149,7 @@ func setChainSpecificConfigDefaultSets() {
 	// With xDai's current maximum of 19 validators then 40 blocks is the maximum possible re-org)
 	// The mainnet default of 50 blocks is ok here
 	xDaiMainnet := fallbackDefaultSet
+	xDaiMainnet.chainType = "XDai"
```

Comment:
Hmm, can we make the types all lower-case? Otherwise the casing can be ambiguous.

Reply:
Would it be worth making it a special type e.g. like this:

```
type  ChainType string
var (
  Arbitrum = ChainType("arbitrum")
  Optimism = ChainType("optimism")
... etc
)

```

Reply:
Maybe. I had considered constant string vars but didn't want to get caught up bike shedding on how to name. I'll try to find a good middle ground.

Reply:
Done.